sample/src/androidTest/java/com/schibsted/spain/barista/sample/RecyclerViewMatcherTest.java

```diff
@@ -0,0 +1,45 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.InstrumentationRegistry;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.assertion.BaristaRecyclerViewAssertions.assertDisplayedAtPosition;
+import static com.schibsted.spain.barista.assertion.BaristaRecyclerViewAssertions.assertRecyclerViewItemCount;
+
+@RunWith(AndroidJUnit4.class)
+public class RecyclerViewMatcherTest {
```

Comment:
We should add some tests in this class to assert that the assertions fail if the condition isn't met.